packages/animations/lib/src/open_container.dart

```diff
@@ -700,6 +676,8 @@ class _OpenContainerRoute<T> extends ModalRoute<T> {
       case AnimationStatus.reverse:
         isInProgress = true;
         break;
+      case null:
```

Comment:
Since we are not doing anything with the null case I think it makes sense to drop this case statement.

Reply:
Dart analyzer gives this warning when `case null` is removed:

```
warning • Missing case clause for 'null' at open_container.dart:682:5 • (missing_enum_constant_in_switch)
```

`case null` is equivalent to `default` here.